test/Microsoft.AspNet.FileProviders.Embedded.Tests/EmbeddedFileProviderTests.cs

```diff
@@ -35,7 +35,7 @@ public void GetFileInfo_ReturnsFilesAtRoot(string filePath)
         {
             // Arrange
             var provider = new EmbeddedFileProvider(GetType().GetTypeInfo().Assembly, Namespace);
-            var expectedFileLength = new FileInfo("File.txt").Length;
```

Comment:
Because that file is not available. There's no way to copy artifacts on publish...